docs-2.0/16.eco-tools/3.nebula-spark-connector.md

```diff
@@ -0,0 +1,35 @@
+# Nebula Spark Connector
+
+Nebula Spark Connector is a Spark connector application for reading and writing Nebula Graph data in Spark standard format. Nebula Spark Connector consists of two parts: Reader and Writer.
+
+* Reader
+  
+  Provide a Spark SQL interface. This interface can be used to read Nebula Graph data. Read one vertex or edge type data at a time, and assemble the result into a Spark DataFrame.
+
+* Writer
+
+  Provide a Spark SQL interface. Use this interface to write data in DataFrame format to Nebula Graph on a strip-by-strip or bulk basis.
+
+For more information, see [Nebula Spark Connector](https://github.com/vesoft-inc/nebula-spark-utils/blob/v2.0.0/nebula-spark-connector/README.md).
+
+## Use cases
+
+Nebula Spark Connector applicable to the following scenarios:
+
+* Migrate data between different Nebula Graph clusters.
+
+* Migrate data between different graph spaces in the same Nebula Graph cluster.
+
+* Migrate data between Nebula Graph and other data sources.
+
+## Benefits
+
+* Provide kinds of connection configuration items, such as timeout period, number of connection retries, number of execution retries, etc.
```

Comment:
1. The subject is Spark Connector, so we should use "Provides" instead of "Provide". The next sentence has the same problem. Or maybe "Supports" is more commonly used in such a case.
2. "Supports multiple connection settings, such as ..."